datalad_dataverse/add_sibling_dataverse.py

```diff
@@ -139,10 +142,10 @@ class AddSiblingDataverse(ValidatedInterface):
             that is capable of storing any number of historical file versions
             using a content hash layout ('annex'|'annex-only'), the 'filetree'
             mode can used.
-            This mode offers a human-readable data organization on the WebDAV
+            This mode offers a human-readable data organization on the dataverse
             remote that matches the file tree of a dataset (branch).
-            However, it can, consequently, only store a single version of each
-            file in the file tree.
+            Note, however, that dataverse comes with restrictions on what file
+            and directory names are possible.
```

Comment:
This will obv. need some more. Guess, should take phrasing from elsewhere

Reply:
Good enough for a start. Thx